database/migrations/2020_01_13_000001_create_order_products_table.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateOrderProductsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('order_product', function (Blueprint $table) {
+            $table->bigIncrements('id');
```

Comment:
this doesn't need a primary key if its a pivot table ?

Reply:
Good catch. I copied this migration from https://github.com/Laracommerce/laracom/blob/5cdddc4bc5aa15b4791799c224892b09d8c6a5d4/database/migrations/2017_06_13_091740_create_order_products_table.php .